pytimeops/read_data.py

```diff
@@ -0,0 +1,20 @@
+import pandas as pd
+import pytimeops
+
+t = range(10)
+
+
+def read_file(filename):
+
+    df = pd.read_csv(filename)
+    list_y = []
+    for i, row in df.iterrows():
+        V = df.iloc[i][5:].values
+        MD = df.iloc[i][1:5].values
```

Comment:
```suggestion
        MD = dict(df.iloc[i][1:5])
```
because we want both the keys and the values for the metadata

Reply:
Done!